.github/workflows/update-fls.yaml

```diff
@@ -0,0 +1,25 @@
+name: Update FLS
+on:
+  schedule:
+    - cron: "05 0 * * *"
+  workflow_dispatch:
+jobs:
+    update-fls:
+      runs-on: ubuntu-latest
+      steps:
+        - name: Check out the repo
+          uses: actions/checkout@v4
+        - name: Install uv
+          uses: astral-sh/setup-uv@v6
+        - name: Update FLS
+          run: "./make.py --update-spec-lock-file"
```

Comment:
The one thing that worries me is whether this would open a PR if there are affected guidelines: https://github.com/rustfoundation/safety-critical-rust-coding-guidelines?tab=readme-ov-file#if-you-need-to-audit-the-difference

I'm assuming that perhaps this is skipping next step depending on the return value of the command? And that in turn handles this edge case?

Reply:
@plaindocs

Reply:
I'll poke at this with a view to checking the normal build first, but not till next week.